interactions/console.py

```diff
@@ -32,29 +32,29 @@ def input_number(question, posible_numbers):  # Zadaje pytanie i zwraca liczbę
                 if number in posible_numbers:
                     return number
                 else:
-                    entry = input(f'Ten numer nie nalerzy do listy. Podaj numer należący do listy: {str(posible_numbers)}')
+                    entry = input(f'Ten numer nie nalerzy do listy. Podaj numer należący do listy: {posible_numbers} ')
             else:
-                entry = input(f'To nie jest numer. Podaj numer należący do listy: {str(posible_numbers)}')
+                entry = input(f'To nie jest numer. Podaj numer należący do listy: {posible_numbers} ')
 
     @staticmethod
-    def input_numer_in_range(question, beginr, endr):  # Zadaje pytanie i oddaje liczbę w danym zakresie
+    def input_numer_in_range(question, begining_of_range, end_of_range):  # Zadaje pytanie i oddaje liczbę w danym zakresie

```

Comment:
number, nie numer

Reply:
Jest number

Reply:
nie jest:

+ ![obraz](https://github.com/wikimody/Quarto-project/assets/150141561/1e31d94a-e585-4682-899a-a0847d399786)

+ po tym jak coś poprawiłeś, to dobrze by było żebyś wcisnął resolve, bo jest dużo tych wątków i nie widać które są aktualne a które już nie